src/AutoRest.CSharp/Mgmt/Decorator/SchemaDetection.cs

```diff
@@ -47,5 +48,23 @@ private static RequestParameter GetBodyParameter(ServiceRequest request, Operati
             }
             throw new Exception($"No body param found! Please add the {operationGroup.Key} to its schema name mapping to readme.md.");
         }
+
+        /// <summary>
+        /// Indicates if the given operation group should skip generation or not.
+        /// If the operation group is marked as "SKIP" in configuration, will return false indicating the corresponding container, data... classes should not be generated.
+        /// </summary>
+        /// <param name="og">Operation group.</param>
+        /// <param name="config">Management plane configuration.</param>
+        /// <returns></returns>
+        public static bool ShouldSkipGeneration(this OperationGroup og, MgmtConfiguration config)
```

Comment:
The original idea we had in the meeting was to add extention methods on `MgmtRestClient`, `ResourceContainer`...
However they don't expose `OperationGroup`, so I cannot get their `Resource`.
That's why I came to the idea to setup lookup methods to get OperationGroup from MgmtRestClient...